python/tempo/tsdf.py

```diff
@@ -315,6 +317,8 @@ def asofJoin(self, right_tsdf, left_prefix=None, right_prefix="right", tsPartiti
     :param tsPartitionVal - value to break up each partition into time brackets
     :param fraction - overlap fraction
     :param skipNulls - whether to skip nulls when joining in values
+    :param sql_join_opt - if set to True, will use standard Spark SQL join if it is estimated to be efficient
+    :param suppress_null_warning - when tsPartitionVal is specified, will collect min of each column and raise warnings about null values, set to True to avoid
     """
```

Comment:
Let's add a comment that we will collect stats when the WARNING level is enabled also.

Reply:
Added the comment as part of a note in the function.